CHANGELOG.md

```diff
@@ -33,6 +33,14 @@
   than mutating it.
 * The method `Store.removeRecord` has been replaced with the method `Store.removeRecords`. This will
   facilitate efficient bulk deletes.
+* The low-level `Box` component no longer applies default flex-related styles. Previously `Box` and
+  `HBox` were functionally identical, given that `Box` declared `display: 'flex'` and the default
+  for flex boxes is `flex-direction: 'column'`. Now `Box` makes no assumptions about flexing its
```

Comment:
isn't the default flex-direction: 'row'?

Reply:
Yes you are correct - fixed.  I always get confused because flex "row" == "lay out children in a row, causing them to appear as, er, columns"